docs/interpretation_of_JSON_Schema_draft_7.md

```diff
@@ -34,21 +35,28 @@ Currently the following `not` model properties are interpreted:
 - boolean `not` schemas are not applied.
 
 ## allOf sub schemas
-`allOf` are a bit different then the other [combination keywords](#Processing-sub-schemas) since it can imply inheritance. 
+`allOf` is a bit different than the other [combination keywords](#Processing-sub-schemas) since it can imply inheritance. 
 
-So dependant on whether the simplify option `allowInheritance` is true or false we interpret it as inheritance or simply merge the models together.
-
-## Determining the type for the model
-To determine the types for the model we use the following interpretation (and in that order):
-- `true` schema infers all model types (`object`, `string`, `number`, `array`, `boolean`, `null`, `integer`).
-- Usage of `type` infers the initial model type.
-- Usage of `properties` infers `object` model type.
-- Usage of `const` infers the constant value as type if schema does not have `type` specified.
-- Usage of `enum` infers the enum values as type if schema does not have `type` specified.
+So dependant on whether the interpreter option `allowInheritance` is true or false we interpret it as inheritance or [merge](#Merging-models) the models.
 
 ## Processing sub schemas
-The following JSON Schema keywords are merged with the already interpreted model:
+The following JSON Schema keywords are [merged](#Merging-models) with the already interpreted model:
 - `oneOf`
 - `anyOf`
 - `then`
 - `else`
+
+
+## Merging models
+Because of the recursive nature of the interpreter (and the nested behavior of JSON Schema) it happens that two models needs to be merged together.
```

Comment:
```suggestion
Because of the recursive nature of the interpreter (and the nested nature of JSON Schema) it happens that two models needs to be merged together.
```